src/styles/content.scss

```diff
@@ -335,4 +335,70 @@ table.os-raise-horizontaltable {
 .os-raise-extrasupport a {
   color: #0A3E5B;
   text-decoration: underline;
-}
\ No newline at end of file
+}
+
+.os-raise-accordion {
```

Comment:
Wasn't sure where to put this comment, but I thought I'd flag something I noticed when navigating via keyboard: I see red bottom borders on FF (I would screenshot, but when I go to do it they go away :upside_down_face: ). On Chrome they seem to be black. Maybe those are defaults and there is some styling we need to specify?

Reply:
:thinking: -- No, that's what I'm doing. Maybe I can screen share when we chat later today. Perhaps it is specific to my environment / build of FF.

Reply:
@P-Gill97 : Just as a follow up from our live convo, I confirmed that on my Mac I see the blue vs red when tabbing across the accordion sections.

Reply:
:thinking: -- I see. Playing around a bit, I notice that in general the focus colors when navigating via keyboard vary across my systems (Linux vs Mac) on other items like math, interactives, etc. Given that, I think if we want to tweak these for consistency, it's probably something we need to look into holistically. I vote we leave this alone so we're consistently inconsistent for the time being :sweat_smile: .

Reply:
@P-Gill97 - Yeah, good call as far as getting input from UX. I do think that if we want to create CSS to make things consistent, it's probably a separate issue we need to spin up so we can identify / address multiple items that I'm finding can vary.